airflow/data/agencies.yml

```diff
@@ -1,5 +1,5 @@
 ac-transit:
-  agency_name: AC Transit
+  agency_name: AC Transit (testing)
```

Comment:
Can this be reverted back to the original?

Reply:
Thank you for catching that and sorry for leaving it in. I had to change the file somehow to test the git hook.